mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule GlassFactoryApi.MixProject do
     [
       app: :glass_factory_api,
       version: "0.1.0",
-      elixir: "~> 1.8",
+      elixir: "~> 1.9",
```

Comment:
Now we need elixir 1.9+ because of `System.fetch_env!\1` used [here](https://github.com/Cendrao/glass_factory_api/pull/17/files#diff-610122b96b2e8892b9b99742443ff1fbR21)